### DISCUSSION TITLE
4/24 (월) 조이 - 계층화 아키텍처(Layered Architecture)

### DISCUSSION THREAD

**Discussion by yeonkkk** [woowacourse-study/2023-presentation-study#9] [발표 피드백]
[계층화 아키텍처](https://github.com/woowacourse-study/2023-presentation-study/blob/main/%EB%A0%88%EB%B2%A82/2%EC%A3%BC%EC%B0%A8/2%ED%8C%80/%5B%EC%A1%B0%EC%9D%B4%5D%EA%B3%84%EC%B8%B5%ED%99%94_%EC%95%84%ED%82%A4%ED%85%8D%EC%B2%98.md)

**Comment by amaran-th** [comment]
- 그림 자료가 좀 더 많았으면 좋았을 것 같다! 글이 많은 느낌...😅
- 아이컨택 자주 해줘잉
- 속도도 적당하고 말 더듬는 것도 없이 차분했다!

**Comment by kwonyj1022** [comment]
* 템포가 조금 더 빨라도 좋을 것 같아요
* 그림이 있어 이해하기 좋았고, 장단점을 같이 적어줘서 좋았습니다.

**Comment by JJ503** [comment]
- 자기소개까지 하다니 완벽하시군요
- 이미지를 통해 예시를 들어주어 좋았고, 실제 현재 미션들과 연관지어 주어 더 잘 이해할 수 있었습니다.

**Comment by jyeost** [comment]
- 모놀리식 애플리케이션을 설명하면서 다른 개념을 말해줬는데 그 내용도 발표자료에 있었으면 좋겟어여
- 발표 자료중에 사진이랑 내용이 좀 다른 부분이 있어서 혼란스러웠습니다 (예를 들면, 사진에 없는데 설명에는 있었다...)
- 오늘은 버퍼링이 조금 있는 것 같았어요 !
- 하지만 늘 깔끔한 발표 =bbb
